plugins/search-backend-module-elasticsearch/config.d.ts

```diff
@@ -20,90 +20,97 @@ export interface Config {
     /**
      * Options for ElasticSearch
      */
-    elasticsearch?:
-      | // elastic = Elastic.co ElasticSearch provider
-      {
-          provider: 'elastic';
-
-          /**
-           * Elastic.co CloudID
-           * See: https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/client-connecting.html#authentication
-           */
-          cloudId: string;
-
-          auth: {
-            username: string;
+    elasticsearch?: {
+      clientOptions?: {
+        ssl?: {
+          rejectUnauthorized?: boolean;
```

Comment:
Super brief doc string on what this controls would be handy (these `config.d.ts` files often end up being the de facto source of truth for config documentation).